sql/mysql-compatibility.md

```diff
@@ -99,6 +99,14 @@ TiDB implements the asynchronous schema changes algorithm in F1. The Data Manipu
 + Rename Table
 + Create Table Like
 
+### Database Administration
+
+Many administrative statements in TiDB work as they do in MySQL, but there are some important differences:
+
++ [`ANALYZE TABLE`](statistics.md#manual-collection) works differently in TiDB than in MySQL, in that it's a relatively lightweight and short-lived operation in MySQL/InnoDB, while in TiDB it completely rebuilds the statistics for a table and can take much longer to complete.
```

Comment:
This is a broken link at the PingCAP website. For link path, please start from the root directory. Two choices:

- From the root directory to the current file: `/sql/statistics.md`
- From the current file to the root directory: `../sql/statistics.md`